client/src/main/java/org/asynchttpclient/DefaultAsyncHttpClient.java

```diff
@@ -112,6 +115,15 @@ public void close() {
                     LOGGER.warn("Unexpected error on HashedWheelTimer close", t);
                 }
             }
+            
+            //see https://github.com/netty/netty/issues/2084#issuecomment-44822314
+            try {
+                ThreadDeathWatcher.awaitInactivity(5, TimeUnit.SECONDS);
```

Comment:
Yes, I guess aligning it with `AsyncHttpClientConfig#getShutdownTimeout` makes absolutely sense. The quietPeriod seems to have different meaning and I guess it is inappropriate at this place.